tests/optimize/test_optimize.py

```diff
@@ -52,11 +51,8 @@ def opt(opt_name):
     if opt_name == "adam":
         return AdamOptimizer(stepsize, beta1=gamma, beta2=delta)
 
-    if opt_name == "roto":
-        return RotosolveOptimizer()
 
-
-@pytest.mark.parametrize("opt_name", ["gd", "moment", "nest", "ada", "rms", "adam", "roto"])
+@pytest.mark.parametrize("opt_name", ["gd", "moment", "nest", "ada", "rms", "adam"])
```

Comment:
The tests for Rotosolve had quite some exceptions already and would require a good bit more because of the differing syntax in `opt.step` that provides the information about the frequency spectra/numbers. I therefore think that the overall readability would suffer from including Rotosolve here, and the dedicated tests should cover all functionalities.

Reply:
:+1: